src/planning.py

```diff
@@ -57,8 +62,58 @@ def sesame_plan(task: Task,
 
     Return a sequence of options, and a dictionary of metrics for this
     run of the planner. Uses the SeSamE strategy: SEarch-and-SAMple
-    planning, then Execution.
+    planning, then Execution. The high-level planner can be either A* or
+    Fast Downward (FD). In the latter case, we allow either optimal mode
+    ("fdopt") or satisficing mode ("fdsat"). With Fast Downward, we can
+    only consider at most one skeleton, and DiscoveredFailures cannot be
```

Comment:
it doesn't really matter, but why can't FD handle discovered failures? isn't all that jazz orthogonal to skeleton generation?

Reply:
okay I see you addressed this and makes sense